# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_redirect_from_script.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_redirect_from_script.js
--- a/netwerk/test/unit/test_redirect_from_script.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_redirect_from_script.js	Tue Feb 12 16:08:25 2019 +0000
@@ -9,7 +9,7 @@
  * made with XMLHttpRequest objects.
  *
  * testViaAsyncOpen() checks that internal redirects occur correctly when made
- * with nsIHTTPChannel.asyncOpen2().
+ * with nsIHTTPChannel.asyncOpen().
  *
  * Both of the above functions tests four requests:
  *
@@ -168,7 +168,7 @@
 
 function makeAsyncTest(uri, headerValue, nextTask)
 {
-  // Make a test to check a redirect that is created with channel.asyncOpen2()
+  // Make a test to check a redirect that is created with channel.asyncOpen()
 
   // Produce a callback function which checks for the presence of headerValue,
   // and then continues to the next async test task
@@ -183,11 +183,11 @@
     nextTask();
   };
 
-  // Produce a function to run an asyncOpen2 test using the above verifier
+  // Produce a function to run an asyncOpen test using the above verifier
   var test = function()
   {
     var chan = make_channel(uri);
-    chan.asyncOpen2(new ChannelListener(verifier));
+    chan.asyncOpen(new ChannelListener(verifier));
   };
   return test;
 }